web/src/components/modal-event-details/index.tsx

```diff
@@ -155,20 +150,20 @@ export const ModalEventDetails: React.FC<IModalEventDetails> = ({
 
   const watchStartTime = watch('startTimeValue');
   const watchEndTime = watch('endTimeValue');
-  const watchAddADestination = watch('addADestination');
   const watchDate = watch('dateValue');
   const watchTitle = watch('title');
 
+  // Note -> this causes following warning:
+  // Warning: Encountered two children with the same key, `Event name`.
+  // Keys should be unique so that components maintain their identity across updates.
+  // Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
+  // Since we can't set the key prop ourselves I don't think this is fixable in an easy way...
   const titleOptions = useMemo(() => {
-    const options = allEvents.map((e: IEvent) => {
-      return {
-        ...e,
-        key: e.id,
-        label: e.title,
-      };
-    });
-
-    return options;
+    return allEvents.map((e: IEvent) => ({
+      ...e,
+      key: e.id, // Does nothing (see above)
```

Comment:
I don't understand. These events come from the API. I had a look at the api and the events are stored in the database, with each event having a random uuid generated ID. How can the ID be the event name?

If you you only get this warning after you copied an event, can't you change the 'copy logic', assigning a different ID to the copied event?

Reply:
It has nothing to do with the id in this case.
The issue is when you create 2 events with the same title, either by copying the event or manually.
The autocomplete component uses the label instead of either the supplied 'id' or 'key' prop to set its keys, so when you don't change the event title when copying an event we end up with the warning.

Reply:
Alright, that is clear. Thank you